filter_lists/default.json

```diff
@@ -110,5 +110,12 @@
         "title": "Brave Android-Specific Rules",
         "format": "Standard",
         "support_url": "https://github.com/brave/adblock-lists"
+    },
+    {
+        "uuid": "b0433353-1144-473c-8e25-19fb035024b7",
+        "url": "https://raw.githubusercontent.com/brave-experiments/sugarcoat-pipeline/main/brave/collated_sugarcoat_rules.txt",
```

Comment:
these rules should live somewhere else, probably https://github.com/brave/adblock-lists/

Reply:
Done

Reply:
And some minor cleanups on the filters in https://github.com/brave/adblock-lists/pull/623

Reply:
I updated the resources here to correspond with https://github.com/brave/adblock-lists/pull/623#issuecomment-893178560. Note that the `dist/resources.json` file is now large enough (~111MB) that Git LFS has to be used for it, which I do in this PR. This will require a change in the URL pointed to by https://github.com/brave/brave-core-crx-packager/blob/master/lib/adBlockRustUtils.js#L12 - if this PR looks good I'll create YAP (yet another PR) to address that.